src/common/featuresToggles.js

```diff
@@ -1,22 +1,14 @@
 import featureToggles from 'feature-toggles';
 import camelCase from 'lodash/camelCase';
 
-import { getApplicationEnvVar } from 'common/injectGlobals';
-
 const featureTogglePrefix = 'REACT_APP_FT';
 
-const isEnabled = (featureName, defaultValue = true) => {
-  const flag = getApplicationEnvVar(featureName);
-
-  // missing feature flags defaults to their default
-  if (typeof flag === 'undefined') return defaultValue;
-
+const isEnabled = flag => {
   // any casing for 'false', false, 0 and '0' are all disabled
   // eslint-disable-next-line eqeqeq
-  if (flag === false || flag.toString().toLowerCase() === 'false' || flag == 0) {
+  if (!flag || flag.toString().toLowerCase() === 'false' || flag == 0) {
```

Comment:
With that change, it would be a little more readable to change `flag == 0` for `flag === '0'` and get rid of `eslint-disable-next-line eqeqeq`.

Reply:
yes, I'll change that